packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -145,13 +166,15 @@ class _LinearProgressIndicatorPainter extends CustomPainter {
     this.value,
     required this.animationValue,
     required this.textDirection,
+    required this.hasRoundStroke,
```

Comment:
I think having a radius for the linear one makes sense. Also, which stroke? The value one? Or the track?

Reply:
`hasRoundIndicator`?

I'll change linear, just tell me what the name should be for the theme to work...

Reply:
See comment above on theming